build.sbt

```diff
@@ -1,17 +1,21 @@
 ThisBuild / tlBaseVersion := "0.23"
 ThisBuild / tlMimaPreviousVersions ++= (0 to 11).map(y => s"0.23.$y").toSet
 ThisBuild / developers := List(
-  tlGitHubDev("rossabaker", "Ross A. Baker")
+  tlGitHubDev("rossabaker", "Ross A. Baker"),
+  tlGitHubDev("henricook", "Henri Cook"),
 )
 
-val Scala213 = "2.13.10"
-ThisBuild / crossScalaVersions := Seq("2.12.17", Scala213)
```

Comment:
Scala 2.12 is still supported in play-json 2.10.1, isn't it a bit early to drop it?

Meanwhile, we can add Scala 3 now?

Reply:
I think that's a mistake on their part, as they looked to be preparing things to be matching Play 2.9's release which drops 2.12. BUT - you're right, until they actually do - we can keep supporting. Will amend and add Scala 3, good call.

Reply:
Amended